pkg/detectors/launchdarkly/launchdarkly_test.go

```diff
@@ -132,6 +129,14 @@ func TestLaunchDarkly_FromChunk(t *testing.T) {
 					t.Fatalf("no raw secret present: \n %+v", got[i])
 				}
 				got[i].Raw = nil
+
+				// Do we expect to be comparing the ExtraData?
```

Comment:
The template detector doesn't include the comparison feature because not all detectors support it. Implementors can choose to test it, but it's not required.